### PR TITLE
small visual fix for works for you render

### PR DESCRIPTION
**Pull request by oxaudo**
@damassi - I think the latest reshuffle brought this issue: https://artsyproduct.atlassian.net/browse/DISCO-489

Seems like the simple fix here and it makes sense. Cause once the data is loaded the content is re rendered.

### CODE REVIEW

File: src/Apps/Artwork/ArtworkApp.tsx [hunk 1]
```diff
@@ -52,22 +52,24 @@ export const ArtworkApp: React.SFC<Props> = props => {
...
         </Col>
       </Row>
-      <Row>
-        <Col sm={8}>
-          <ArtistInfo artistID={props.artwork.artist.id} />
-        </Col>
-      </Row>
-
-      {typeof window !== "undefined" && (
+      {props.artwork.artist && (
```
Comment (oxaudo) [inline src/Apps/Artwork/ArtworkApp.tsx]: Also looked at some Cultural maker artworks - those don't have artists and will crash here.

Technically I think there should be a test for this stuff but not sure if we have any of those yet for the top level page.

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 8dac383bfaa2b84cd5759b26b7556e86f7a6a7c4

https://deploy-preview-1504--artsy-reaction.netlify.com

File: src/Apps/Artwork/ArtworkApp.tsx (hunk 1 above)
Comment (damassi) [reply src/Apps/Artwork/ArtworkApp.tsx]: We're just in the cleanup phase now, all of this stuff is gonna be moved around. Not ready to be tested (but will be tested soon).

**Comment by damassi** [comment]
👍

**Pull request merged**